### PR TITLE
Create mobile navigation

### PR DESCRIPTION
**Pull request by MahdiRezaeiDev**
**### Portfolio project: creating mobile navigation**
In this branch, we have made the following changes.

- Added bars and close icons for mobile nav.
- Added javascript code to navigate between the above two icons when it should.
- Added proper styling for mobile navigation.
- Added smooth scroll behavior to scrolling to sections.
- Updated README.md file.

### CODE REVIEW

File: src/js/main.js [hunk 1]
```diff
@@ -1,9 +1,30 @@
...
+});
+
+const navIcon = document.querySelector('.mobile-menu');
+const mainNav = document.querySelector('.main-nav');
+
+navIcon.addEventListener('click', () => {
+  mainNav.style.height = '100%';
+  mainNav.classList.toggle('open');
+  if (mainNav.classList.contains('open')) {
+    navIcon.src = './src/images/close-icon.svg';
```
Comment (FrederickMih) [inline src/js/main.js:17]: - [x] This icon does not load immediately when clicked on the mobile menu options. I noticed that I needed to always refresh the page for it to load again. I think this is not normal. When the user clicks (or taps) any option of the mobile menu, the mobile menu disappears and not the hamburger button. Kindly fix this issue.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- No Linter error :heavy_check_mark: 
- PR has a title and summary :heavy_check_mark: 

## Required Changes ♻️

- [ ]  Based on the requirements of this project, we are going to setup up both [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML/CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) Linters in this project repository.



_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/js/main.js (hunk 1 above)
Comment (MahdiRezaeiDev) [reply src/js/main.js:17]: Thank you fro your feedback, I have fixed the mentioned issue

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @MahdiSohaily , 

Great work on making the changes requested by a previous reviewer 👏 🔥 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



## Top highlight 🎯

- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 


 ## Required Changes ♻️

- [ ] As suggested by the previous reviewer, kindly ensure you add the javascript linter config file (i,e `.eslintrc.json` ) as the file is currently missing in your project. 

Please follow this[ link](https://github.com/microverseinc/linters-config/tree/master/html-css-js) carefully to set it up 👍🏽 

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
### Hi Team , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**